### PR TITLE
Add third-party plugin mill-explicit-deps

### PR DESCRIPTION
**Pull request by kierendavies**

### CODE REVIEW

File: docs/antora/modules/ROOT/pages/Thirdparty_Plugins.adoc [hunk 1]
```diff
@@ -120,6 +120,40 @@ Optionally, you can specify the ensime server version using the –server flag l
...
+
+A plugin that checks that `ivyDeps` and `ivyCompileDeps` accurately reflect the direct dependencies of your source code.
+
+Project home: https://github.com/kierendavies/mill-explicit-deps
+
+=== Quickstart
+
+_build.sc_:
+
+[source,scala]
```
Comment (lefou) [inline docs/antora/modules/ROOT/pages/Thirdparty_Plugins.adoc:133]: Optional title start with a `.` in Asciidoctor.

```suggestion
.`build.sc`
[source,scala]
```

**Review by lefou — COMMENTED** [review]
Nice addition. Thank you!

**Comment by lefou** [comment]
@kierendavies I noticed that you refer to a version `0.1.0` in the quickstart, but the repo does not have any tags or releases listed. Is your plugin already published?

**Comment by kierendavies** [comment]
@lefou I didn't push the tag 🤦 but yes it's published https://search.maven.org/artifact/io.github.kierendavies/mill-explicit-deps_mill0.10_2.13

Thanks for the speedy review and merge!

**Pull request merged**